diff --git a/src/daemon.vala b/src/daemon.vala
index 5b55bf1..8d81997 100644
--- a/src/daemon.vala
+++ b/src/daemon.vala
@@ -696,7 +696,7 @@ namespace Pamac {
 				bool authorized = check_authorization.end (res);
 				if (authorized) {
 					try {
-						new Thread<int>.try ("snap_trans_run", () => {
+						new Thread<int>.try ("flatpak_trans_run", () => {
 							AtomicInt.inc (ref running_threads);
 							bool success = flatpak_plugin.trans_run (sender, to_install_copy, to_remove_copy, to_upgrade_copy);
 							flatpak_trans_run_finished (sender, success);
